### PR TITLE
[WIP] Update test order to fix flake in operator tests

### PR DESCRIPTION
**Pull request by jeff-phillips-18**

### CODE REVIEW

File: frontend/integration-tests/protractor.conf.ts [hunk 1]
```diff
@@ -105,23 +105,23 @@ export const config: Config = {
...
       'tests/secrets.scenario.ts',
-      'tests/olm/**/*.scenario.ts',
       'tests/service-catalog/**/*.scenario.ts',
       'tests/filter.scenario.ts',
       'tests/modal-annotations.scenario.ts',
       'tests/source-to-image.scenario.ts',
       'tests/deploy-image.scenario.ts',
       'tests/operator-hub/operator-hub.scenario.ts',
-      'tests/developer-catalog.scenario.ts'],
+      'tests/developer-catalog.scenario.ts',
```
Comment (benjaminapetersen) [inline frontend/integration-tests/protractor.conf.ts]: Should ordering matter?  Does that reveal a dependency between some of the tests?

Comment (benjaminapetersen) [reply frontend/integration-tests/protractor.conf.ts]: Wondering if we need a `FIXME` for isolation?

Comment (jeff-phillips-18) [reply frontend/integration-tests/protractor.conf.ts]: I haven't made that determination yet. Just a thought that I'm testing out.

**Pull request closed without merging**